.ci/linux-steps.yaml

```
@@ -55,7 +56,12 @@ steps:
       export GOPATH=$PWD/src/mlpack/bindings/go
       go get -u -t gonum.org/v1/gonum/...
     fi
-    cmake $(CMakeArgs) ..
+    if [ "$(binding)" == "python" ]; then
+      export PYBIN=$(which python)
+      cmake $(CMakeArgs) -DPYTHON_EXECUTABLE=$PYBIN ..
+    else
+      cmake $(CMakeArgs) ..
+    fi
```

I think you could *always* specify `-DPYTHON_EXECUTABLE`; if `BUILD_PYTHON_BINDINGS` is `OFF`, it should just be ignored:

```suggestion
cmake $(CMakeArgs) -DPYTHON_EXECUTABLE=`which python` ..
```

I think also that could avoid the need for `PYBIN`, unless I missed some other functionality of `PYBIN`.